src/peer.ts

```
@@ -0,0 +1,1178 @@
+  return peerPubkeys;
+}
+
+/**
+ * Simple function to ping peers and get online peer pubkeys (legacy compatibility)
+ * @deprecated Use PeerManager.pingPeers() or pingPeersInternal() instead
+ */
+export async function pingPeers(node: BifrostNode, timeout = DEFAULT_PING_TIMEOUT): Promise<string[]> {
+  try {
+    // Use Bifrost's native ping functionality for backward compatibility
+    const result = await (node.req as any).ping();
+    if (result && result.ok) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)